tests/playwright/page-object-model/search-page.ts

```diff
@@ -27,25 +37,40 @@ export class SearchPage {
   }
 
   async goToSearchFor (searchTerm: string): Promise<void> {
-    this.searchForm.currentSearchTerm = searchTerm
+    this.currentSearch.term = searchTerm
     await this.page.goto(`/search/?keywords=${searchTerm}`)
   }
 
+  async goToPageWithResults (): Promise<void> {
+    await this.goToSearchFor('mary')
+  }
+
+  async goToPageWithNoResults (): Promise<void> {
+    await this.goToSearchFor('non')
```

Comment:
We should make this a little longer and definitely invalid so there's no chance future test data will match it